Kitodo/src/main/resources/xslt/dfg-viewer.xsl

```diff
@@ -11,13 +11,136 @@
  * GPL3-License.txt file that was distributed with this source code.
  *
 -->
-<xsl:stylesheet version="2.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+<xsl:stylesheet version="2.0"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:mets="http://www.loc.gov/METS/"
+                xmlns:xlink="http://www.w3.org/1999/xlink"
```

Comment:
```suggestion
```
XLink namespace is not used in this xslt file and can be removed from the declaration.